.github/comment.yml

```diff
@@ -39,7 +63,19 @@
 - rule:
     type: label
     label: NewApiVersionRequired
-    onLabeledComments: "The automation detected this Pull Request introduces breaking changes to an existing API version and hence it added the <code>NewApiVersionRequired</code> label. This means you cannot proceed with merging this PR until you complete one of the following action items:<br/><br/>- A) Submit a new PR instead of this one, or modify this PR, so that it introduces a new API version instead of introducing breaking changes to an existing API version. The automation will remove the label once it detects there are no more breaking changes.<br/>- B) OR you can request an approval of the breaking changes, get it reviewed, and approved. The reviewer will add <code>Approved-BreakingChange</code> label if they approve.<br/><br/>For additional guidance, please see https://aka.ms/NewApiVersionRequired<br/>  "
+    onLabeledComments: >-
+      The automation detected that this PR introduces breaking changes to an existing API version and hence it added the `NewApiVersionRequired` label. This means you cannot proceed with merging this PR until you complete one of the following action items:
+      <br/><br/>
+      **ACTION ITEM** for you:
+      <br/><br/>
+      - A) Submit a new PR instead of this one, or modify this PR, so that it introduces a new API version instead of introducing breaking changes to an existing API version. The automation will remove the label once it detects there are no more breaking changes.<br/>
+      - B) OR you can request an approval of the breaking changes, get it reviewed, and approved. The reviewer will add `Approved-BreakingChange` label if they approve.
+      <br/><br/>
+      For more information, please refer to https://aka.ms/specprreview
+    onLabeledRemoveLabels:
+        - WaitForARMFeedback
+    onLabeledAddLabels:
+        - ARMChangesRequested
```

Comment:
I think this is wrong if `NewApiVersionRequired` can also happen for data-plane PRs, and I think it can, based on the [BreakingChangeRules.yml config](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/azure-rest-api-specs-pipeline/config/BreakingChangeRules.yml&version=GC0a9aa5e599c77ebca04b230ccc0933ae68dab3fa&line=1&lineStartColumn=3&lineEndColumn=3&_a=contents).

If this is the case, then I could constrain this rule to work only when `ARMReview` label is present. This label won't be present if the PR author forgot to check relevant checkmark in the PR, but if so, this could be handled be appropriate rule that reacts to checking the checkmark when `NewApiVersionRequired` is present.

Reply:
OK, I fixed this by splitting the rule in two, but I don't think this will work, because I think source expects only one occurrence of (`type`, `label`) pair, as can be seen by 

[`getMatchedRuleByLabel`](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=333&lineStartColumn=11&lineEndColumn=32&_a=contents) returning first match to [the caller `handlePullRequestLabeled`](https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_git/openapi-alps?path=private/openapi-kebab/src/bots/workflow/workflowBot.ts&version=GC9df1258de7cdcbd322cfc2ff15f4ef4f96998f9d&line=224&lineStartColumn=23&lineEndColumn=44&_a=contents).

Reply:
Currently we are considering making all the initial labelling to happen via the CI checks / summary task, and make the workflow bot deal only with things that would happen after the PR was submitted and all initial CI checks ran. In such case, any handling of `NewApiVersionRequired` and relevant comments could be done in TypeScript, not in workflow-bot rules.